functions.php

```diff
@@ -50,16 +50,15 @@ function getCoordinatesForAddress($address) {
     $coordinates = new Coordinates();
 
     if (strlen($address) > 0) {
-        $map_details_response = get($GLOBALS['google_maps_endpoint']
+        $map_details_response = get($GLOBALS['maps_endpoint']
             . "&address="
             . urlencode($address)
-            . (isset($GLOBALS['location_lookup_bias']) ? "&components=" . urlencode($GLOBALS['location_lookup_bias']) : ""));
+            . (isset($GLOBALS['location_lookup_bias']) ? "&country=" . urlencode($GLOBALS['location_lookup_bias']) : ""));
```

Comment:
this would be better if it was "&countrycodes=" as this would force the results to be a specific country better